buildSrc/src/main/groovy/org/elasticsearch/gradle/test/RestTestPlugin.groovy

```diff
@@ -34,6 +34,7 @@ class RestTestPlugin implements Plugin<Project> {
         integTest.configure {
             classpath = project.sourceSets.test.runtimeClasspath
             testClassesDir project.sourceSets.test.output.classesDir
+            cluster.distribution = 'zip' // rest tests should run with the real zip
```

Comment:
The Java tests run against `integ-test-zip` but the REST tests run against `zip`. On the face of it it seems pathological but it makes sense on a deeper level because plugins shouldn't see the modules classpath anyway. Can they be runtime dependencies or something? I dunno. I'm trying to square "its ok to have a groovy script in a plugin's rest tests but not in its java tests".